nestkernel/eprop_archiving_node_impl.h

```diff
@@ -50,17 +50,17 @@ EpropArchivingNode< HistEntryT >::EpropArchivingNode( const EpropArchivingNode&
 
 template < typename HistEntryT >
 void
-EpropArchivingNode< HistEntryT >::register_eprop_connection()
+EpropArchivingNode< HistEntryT >::register_eprop_connection( const bool is_bsshslm_2020_model )
 {
   ++eprop_indegree_;
 
-  const long shift = get_shift();
+  const long t_first_entry = is_bsshslm_2020_model ? get_shift() : -delay_rec_out_;
```

Comment:
I am not happy with these case distinctions. One can in principle solve this by overriding `register_eprop_connection()` in the concrete classes, but that requires doing it twice, once for the recurrent and once for the readout class, which is not neat. Need to think about this a little more.